ext-src/packages/gradle/GradleUtils.ts

```diff
@@ -31,6 +31,7 @@ export class GradleUtils {
   private readonly GRADLEW = `./gradlew`;
   private readonly GRADLEW_BAT = `gradlew.bat`;
   private readonly gradleArguments = `dependencies --configuration runtimeClasspath`;
+  private readonly gradleArgumentsTest = `dependencies --configuration testRuntimeClasspath`;
```

Comment:
For backend projects I think that's ok, but I'm wondering if that works for Android projects 🤔

The configuration holding "production" dependencies in Android projects is `releaseRuntimeClasspath` so I'm not sure if `testRuntimeClasspath` holds the same for testing.

Reply:
Hey @guillermo-varela. The reason I put in `testRuntimeClasspath` was that that's the option toggled by the "Nexus explorer: Include Dev" setting according to the `parseGradleDependencyTree` function in `GradleUtils.ts`. So I guess it depends on whether there's some more complex logic required overall to handle the different cases.

Reply:
Since `releaseRuntimeClasspath` is not being included I guess it's safe to assume Android projects are not a priority at the moment (considering that Android Studio is the IDE to use for those), so my comment can be taken as just as a FYI rather than a request for change :)